io.github.theoden8.orange_unofficial.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: io.github.theoden8.orange_unofficial
```

Comment:
So why is this id unrelated to `https://github.com/biolab/orange3/` or why did you change from `com.orangedatamining`?